### DISCUSSION TITLE
memory use seems high

### DISCUSSION THREAD

**Discussion by ak1dd** [Ross-Patterson/Portals-Desktop-Organization#204] [General]
When checking process memory use with Portals 3.0.3.3 on Windows 11 (fully updated as of today) I see memory usage at 139.5-207.4MB

Fences doesn't have a process that I can identify and from what I've read it is injected into explorer, but if I disable and restart explorer the difference running is only about 3-5MB.  Reading the SD forums that seems to be what others see also.

**Comment by PhoenixR1s1ng** [comment]
I am see about a 100MB difference with explorer.exe when fences 4 is installed versus when I completely uninstall it. portals is peaking at about 100 - 135MB maybe just a smidge higher but pretty comparable.

edit - it may be actually a 50mb difference with explore.exe and fences. It is hard to calculate for sure.
